pkgs/development/python-modules/pyft232/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyserial
+, pyusb
+}:
+
+buildPythonPackage rec {
+  pname = "pyft232";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "lsgunth";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-t9CKTZCUro9Fn7k3WwCZVoH8VS3KZrpu9WSXEN6sPl0=";
+  };
+
+  propagatedBuildInputs = [
+    pyserial
+    pyusb
+  ];
+
+  doCheck = false;
```

Comment:
no tests ?

Reply:
Hmm I could emulate the test here, though additional `libftd2xx` package is needed. Looks like this one is closed source and might be proprietary. Not sure if we can package this?

Reply:
it's ok to disable tests, just good to leave a comment explaining the reason